packages/apollo-graphql/src/transforms.ts

```diff
@@ -94,13 +95,37 @@ function sorted<T>(
   return undefined;
 }
 
+// Predicate for filtering and type checking operation definitions (query, mutation, subscription)
+// Similar to graphql's isExecutableDefinitionNode
+function isOperationDefinitionNode(
```

Comment:
`isOperationDefinitionNode` and  `isFragmentDefinitionNode` should probably be moved to a separate `utilities.ts` (and submitted upstream to `graphql-js` at some point).

Reply:
I agree! These are no longer needed due to a different approach, but I've wanted these elsewhere. I'll submit a PR, maybe this weekend 😄

Reply:
Created a `utilities` file anyway, and moved one of our functions there: 1b70cd365f07ca80d218738f9e6156e0c0390b84